fontscan/fontmap.go

```diff
@@ -314,6 +314,26 @@ func (fm *FontMap) FontMetadata(ft font.Font) (family string, aspect meta.Aspect
 	return item.Family, item.Aspect
 }
 
+// FindSystemFont looks for a system font with the given [family],
+// returning the first match, or false is no one is found.
+//
+// User added fonts are ignored, and the [FontMap] must have been
+// initialized with [UseSystemFonts] or this method will always return false.
+//
+// Family names are compared through [meta.Normalize].
+func (fm *FontMap) FindSystemFont(family string) (Location, bool) {
```

Comment:
Just a question - is the not found case better as "<undefined> false" or "nil" to avoid the second return parameter?

Reply:
No strong here, but, since Location is a struct, that would force the user to check Location.File against the empty string, which I find a bit akward. Also, the current signature matches the general map lookup syntax in Go.

Reply:
I like matching the semantics of map access for this. An explicit value signalling whether anything was found.